src/main/java/com/thebois/models/beings/IBeing.java

```diff
@@ -10,10 +11,24 @@
     /**
      * Gets the current location.
      *
-     * @return The current location.
+     * @return The current location
```

Comment:
What does the codestyle want? I guess either is ok?

Reply:
Yeah, that's something I've noticed that we all use different standards for.

I see them as *sentence fragments* that don't need to be a complete sentence in and of themselves, as opposed to the actual javadoc comment for the method itself. But I've seen code with periods as well. I've also seen some without capital first letter and without period. We should probably pick one and stick to it.

I'll look into adding this to the checkstyle!

Reply:
Doesn't seem to be an option in checkstyle. Maybe with some regex magic, idk.

Anyways, added it as a discussion point for Friday.